charts/eol-exporter/values.yaml

```diff
@@ -0,0 +1,157 @@
+---
+eol-exporter:
+  enabled: true
+  namespaceOverride: ''
+  applicationName: eol-exporter
+  externalSecrets:
+    enabled: false
+  deployment:
+    enabled: true
+    image:
+      # Image is built and uploaded into private ECR based on source code
+      # Refer to the Readme to know how to push new public tags:
+      # https://github.com/saritasa-nest/saritasa-devops-tools-eol-exporter/blob/main/config.yaml.example
+      # Public image is available on:
+      # https://hub.docker.com/r/saritasallc/eol-exporter/tags
+      repository: saritasallc/eol-exporter
+      tag: prod-843dabc
```

Comment:
retag it using semantic versioning, i.e. 0.0.1 etc

Reply:
Done. Chart version will be updated when PR is approved